src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -372,7 +372,7 @@ final class OnboardingInfoViewController: UIViewController {
 
 extension OnboardingInfoViewController: UITextViewDelegate {
 	func textView(_ textView: UITextView, shouldInteractWith url: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
-		WebPageHelper.openSafari(withUrl: url, from: self)
+		WebPageHelper.open(withUrl: url, from: self)
```

Comment:
Could you please double check that this change has actually any effect in practice, i.e. that links are really opened in Onboarding?

On my device they're not, and I suspect it will be the same on all devices since ```dataDetectorTypes``` were removed from ```HtmlTextView```. 

Furthermore, based on concerns that Marc-Peter already raised, I would also suggest to align with PO on this, as apparently this was discouraged by PO/Design in the original implementation.

Reply:
Just checked the ticket. Based on the ticket, there's no mention of Onboarding at all. Hence I would suggest rather to don't touch Onboarding here at all, since concerns were raised, and basically no-one asked for changes there.

Reply:
I have changed it back so that the OnboardingViewController only uses the functionality for opening webpages like it was using it before (no switch statement, just plain old method).